packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/TreeItem.module.scss

```diff
@@ -0,0 +1,162 @@
+@import '@deephaven/components/scss/custom.scss';
+
+.Wrapper {
+  list-style: none;
+  box-sizing: border-box;
+  padding-left: var(--spacing);
```

Comment:
Should be a $spacer-N?

Reply:
Not used any more. Was initially the way the example set the indent for each item. Instead I added depth lines so this is now removed